snmp-observ-lib/panels/fleet.libsonnet

```diff
@@ -0,0 +1,129 @@
+
+    errors:
+      signals.fleetInterface.networkInErrorsPerSec.withTopK().asTimeSeries('Network errors')
+      + signals.fleetInterface.networkOutErrorsPerSec.withTopK().asPanelMixin()
+      + commonlib.panels.network.timeSeries.errors.stylize()
+      + commonlib.panels.network.timeSeries.errors.withDataLink(this.config.instanceLabels, this.grafana.dashboards['snmp-overview.json'].uid)
+      + commonlib.panels.network.timeSeries.errors.withNegateOutPackets()
+      + g.panel.timeSeries.queryOptions.withInterval(this.config.minInterval),
+
+    dropped:
+      signals.fleetInterface.networkInDroppedPerSec.withTopK().asTimeSeries('Packets dropped')
+      + signals.fleetInterface.networkOutDroppedPerSec.withTopK().asPanelMixin()
+      + signals.fleetInterface.ifInUnknownProtos.withTopK().asPanelMixin()
+      + commonlib.panels.network.timeSeries.dropped.stylize()
+      + commonlib.panels.network.timeSeries.dropped.withDataLink(this.config.instanceLabels, this.grafana.dashboards['snmp-overview.json'].uid)
+      + commonlib.panels.network.timeSeries.dropped.withNegateOutPackets()
+      + g.panel.timeSeries.queryOptions.withInterval(this.config.minInterval),
+
+    packetsUnicast:
+      signals.fleetInterface.networkInUnicastPacketsPerSec.withTopK().asTimeSeries('Unicast')
+      + signals.fleetInterface.networkOutUnicastPacketsPerSec.withTopK().asPanelMixin()
+      + commonlib.panels.network.timeSeries.unicast.stylize()
+      + commonlib.panels.network.timeSeries.unicast.withDataLink(this.config.instanceLabels, this.grafana.dashboards['snmp-overview.json'].uid)
+      + commonlib.panels.network.timeSeries.unicast.withNegateOutPackets()
+      + g.panel.timeSeries.queryOptions.withInterval(this.config.minInterval),
+
+    packetsBroadcast:
+      signals.fleetInterface.networkInBroadcastPacketsPerSec.withTopK().asTimeSeries('Broadcast')
+      + signals.fleetInterface.networkOutBroadcastPacketsPerSec.withTopK().asPanelMixin()
+      + commonlib.panels.network.timeSeries.broadcast.stylize()
+      + commonlib.panels.network.timeSeries.broadcast.withDataLink(this.config.instanceLabels, this.grafana.dashboards['snmp-overview.json'].uid)
+      + commonlib.panels.network.timeSeries.broadcast.withNegateOutPackets()
+      + g.panel.timeSeries.queryOptions.withInterval(this.config.minInterval),
+
+    packetsMulticast:
+      signals.fleetInterface.networkInMulticastPacketsPerSec.withTopK().asTimeSeries('Multicast')
+      + signals.fleetInterface.networkOutMulticastPacketsPerSec.withTopK().asPanelMixin()
+      + commonlib.panels.network.timeSeries.multicast.stylize()
+      + commonlib.panels.network.timeSeries.multicast.withDataLink(this.config.instanceLabels, this.grafana.dashboards['snmp-overview.json'].uid)
+      + commonlib.panels.network.timeSeries.multicast.withNegateOutPackets()
+      + g.panel.timeSeries.queryOptions.withInterval(this.config.minInterval),
```

Comment:
Based on the screenshot I think these three panels would benefit from a table based legend. Thoughts?

Reply:
it would make panels narrower if table is moved to the right, not so great IMHO. 
And since there are no calculations in the legend, it wouldn't make a lot of sense if changing the legend type to table and keeping at the bottom.
I would keep it as is for now.

Reply:
That's fair, something to keep an eye as we see more busy deployments then!